partcad/src/partcad/part_factory_alias.py

```
@@ -20,36 +20,33 @@
 
             if "source" in config:
                 self.source_part_name = config["source"]
             else:
-                self.source_assembly_name = config["name"]
-                if not "project" in config:
-                    raise Exception(
-                        "Alias needs either the source part name or the source project name"
-                    )
+                self.source_part_name = config["name"]
+                if "project" not in config and "package" not in config:
+                    raise Exception("Alias needs either the source part name or the source project name")
```

Ruff (0.8.2)

Create your own exception

(TRY002)